pages/components/ImageList.js

```diff
@@ -0,0 +1,51 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+const ImageWrapper = styled.div`
+  width: auto;
+  height: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: 10px;
+  margin-right: 10px;
+  border: 3px solid #ccc;
+`;
+
+const ImageListWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+  margin: 20px;
+`;
+
+const ImageList = () => {
+  const images = Array.from(
+    { length: 20 },
+    (_, i) => `/../public/Images/image${i + 1}.png`
```

Comment:
You need to generate a url that is accessible from the web and not just from within your project. Currently, the images aren't visible on Vercel's [deployment](https://capstone-project-2f43cc04n-ctgitwhiz.vercel.app/).

Reply:
I thought this is the way to start? Since i should implement something like "cloudinary" at the end...?